### PR TITLE
added purchase order

### PR DESCRIPTION
**Pull request by geekdeepak**
added basic purchase order model

### CODE REVIEW

File: lib/xeroizer/version.rb [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module Xeroizer
-  VERSION = "2.16.4".freeze
+  VERSION = "2.16.5".freeze
```
Comment (CloCkWeRX) [inline lib/xeroizer/version.rb]: Don't worry about bumping this

File: lib/xeroizer/models/purchase_order.rb [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      string  :branding_theme_id 
+      string  :status 
+      string  :line_amount_types
+      string  :sub_total
+      string  :total_tax
+      string  :total 
+      date    :updated_date_UTC
+      boolean :has_attachments
+
+      # decimal :unit_price
```
Comment (CloCkWeRX) [inline lib/xeroizer/models/purchase_order.rb]: Safe to kill off dead code

**Comment by waynerobinson** [comment]
If you could make the changes @CloCkWeRX suggested, I'd be happy to merge and release. :)

**Comment by geekdeepak** [comment]
@waynerobinson  done.

**Comment by geekdeepak** [comment]
i am waiting for you to approve the PR. Please check and review, let me know if anything else is required.

**Comment by kitebuggy** [comment]
Awesome gem.  The addition of purchase orders would make it perfect for my needs.  What's holding up this PR?  Anything I can help with?

**Comment by brancusi** [comment]
Excellent, would love to see this merged as well.

**Comment by nnc** [comment]
We've been using this in production for about a month now. No issues at all. So yeah, a merge would be nice.

**Comment by kitebuggy** [comment]
Thanks, Wayne - much appreciated!

**Comment by kitebuggy** [comment]
Presumably the version number now needs a bump?

**Comment by charlietarr1** [comment]
Hi Wayne - thanks for creating this gem. I am using version 2.16.4 but am not able to access Purchase Orders as part of a private application. From looking through issues / commits, it seems this feature has been added. Can you please confirm? Thanks

**Comment by nnc** [comment]
@charlietarr1 this has been merged after 2.16.4 was released, and no new versions of gem have been released since then, so you'll have to pull xeroizer from github until new gem version is released by @waynerobinson.

```ruby
# Gemfile
gem 'xeroizer', github: 'waynerobinson/xeroizer'
```

**Comment by charlietarr1** [comment]
Thanks very much - all working!! A Big Help!!

**Pull request merged**